Sample.java

```diff
@@ -1,7 +1,73 @@
-// Time Complexity :
-// Space Complexity :
-// Did this code successfully run on Leetcode :
-// Any problem you faced while coding this :
+// Time Complexity : O(n^2) where n is the number of decisions made at each level
```

Comment:
Time complexity will also be O(M*N) where M is amount and N is the coins, which is matrix traversal